include/ccpp_adapt.h

```diff
@@ -0,0 +1,63 @@
+
```

Comment:
Is there a more expressive name for this header file?

Reply:
renamed to ccpp_2020_compatibility_layer.h